src/pages/AboutUs/Details.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import pic from './pic.png';
+import { MDBRow, MDBCol } from 'mdbreact';
+import { useTranslation } from 'react-i18next';
+
+function Details() {
+  const { t } = useTranslation();
+
+  const aboutPageTranslation = {
+    headerTitle: t('aboutPage.aboutSection.aboutUs'),
+    text: t('aboutPage.aboutSection.aboutUsText'),
+    ReadMore: t('aboutPage.aboutSection.ReadMoreButton'),
+  };
```

Comment:
you can receive every thing one time by extracting an object from the translation 
```suggestion
  const aboutSectionTranslation = t('aboutPage.aboutSection', {returnObject : true})
   ```

then you can the keys in the translation.json file and make it more meaningful and use it like this `aboutSectionTranslation.aboutUsText`